docs/source/transformers/default-values.rst

```diff
@@ -1,14 +1,15 @@
 Default values support
 ======================
 
-Chimney respects case classes' default values as a possible target
+Chimney allows case classes' default values as a possible target
 field value source.
 
-Automatic value provision
--------------------------
+Enabling default values in generated transformer
```

Comment:
I would add warning mentioning that prior to 0.7.0 default values processing was enabled automatically.

Reply:
Done